ui/raidboss/data/06-ew/raid/p11s.ts

```diff
@@ -2,13 +2,36 @@ import Outputs from '../../../../../resources/outputs';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
 export interface Data extends RaidbossData {
+  decOffset?: number;
   lightDarkDebuff: { [name: string]: 'light' | 'dark' };
   lightDarkBuddy: { [name: string]: string };
   lightDarkTether: { [name: string]: 'near' | 'far' };
+  cylinderValue?: number;
+  numCylinders?: number;
 }
+
+const headmarkers = {
+  // P11N uses, and the P11S VFX looks like, 0x00DA for Dike, but using that value gives negative results for the Arcane Cylinder headmarkers
+  // 0x01DB derived by using the P11N Styx 0x0131 headmarker and calculating backwards for Dike
+  // 0x01DB is used in DRS Trinity Seeker for Earthshaker markers, which look and behave nothing like Dike
+  dike: '01DB', // tankbuster
+  styx: '0131', // multi-hit stack, currently unused
+  orangeCW: '009C', // orange clockwise rotation
+  blueCCW: '009D', // blue counterclockwise rotation
+} as const;
```

Comment:
```suggestion
const headmarkers = {
  // vfx/lockon/eff/tank_lockon04_7sk1.avfx
  dike: '01DB', // tankbuster
  // vfx/lockon/eff/com_share4a1.avfx
  styx: '0131', // multi-hit stack, currently unused
  // vfx/lockon/eff/m0515_turning_right01c.avfx
  orangeCW: '009C', // orange clockwise rotation
  // vfx/lockon/eff/m0515_turning_left01c.avfx
  blueCCW: '009D', // blue counterclockwise rotation
} as const;
```

`01DB` in DRS is a tether value and not a headmarker value, for what it's worth.  Your values all look correct to me.  Here's some comments with the game path to the LockOn vfx.